resources/cluster-essentials/charts/pod-preset/values.yaml

```diff
@@ -0,0 +1,19 @@
+webhook:
+  replicaCount: 1
+  image:
+    repository: mszostok/service-catalog-admission-webhook
```

Comment:
need to be clarified where this image should be placed

Reply:
as I wrote in the description, this is a temporary solution until we will be able to download the official ones